packages/ember-metal/lib/meta.js

```diff
@@ -46,6 +47,24 @@ let members = {
   tag: ownCustomObject
 };
 
+// This is less than ideal, but `runInDebug` is
+// not populated by the time we need it here. So
+// we create a function that has the same signature
+// as the real `runInDebug` so that it is stripped
+// from the production build. Unfornately, this
+// function will remain in production build.
+let runInDebug = (fn) => fn();
```

Comment:
@rwjblue this is the circular race that makes part of ember metal things not able to use debug stuff. What I was working toward moving it so metal imports ember debug.